v2/guestbook/main.go

```diff
@@ -267,6 +270,24 @@ func findRedisURL() string {
 	return ""
 }
 
+//getDummyTone is a dummy version for the tone anylyzer
```

Comment:
Can you add some comments here to explain the thinking behind the HasPrefix checks? Meaning, why do you assume a word that starts with "s" is happy? And how does this work when you look for 's' on 275 and 281?

Reply:
hmmm.. I just want to return something to simulate tone analyzer which can show we call our version2 guest book. 

Line 281 is a typo, I should use other beginning instead of "s"， eg: "w"-->worry

Reply:
could we do it based on certain words?  Seems hard to be based on a prefix...

Also, I'd prefer return some indication that we are using dummy tone to generate the tone so we know it is not watson that is so dumb. :)